ee/modules/110-istio/hooks/discovery_istiod_health.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright 2022 Flant JSC
+Licensed under the Deckhouse Platform Enterprise Edition (EE) license. See https://github.com/deckhouse/deckhouse/blob/main/ee/LICENSE
+*/
+
+package hooks
+
+import (
+	"github.com/flant/addon-operator/pkg/module_manager/go_hook"
+	"github.com/flant/addon-operator/sdk"
+	"github.com/flant/shell-operator/pkg/kube/object_patch"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	"github.com/deckhouse/deckhouse/ee/modules/110-istio/hooks/internal"
+)
+
+const isGlobalRevisionIstiodReadyPath = "istio.internal.isGlobalRevisionIstiodReady"
+
+type istiodPod struct {
+	Name     string
+	Revision string
+	Phase    v1.PodPhase
+}
+
+var _ = sdk.RegisterFunc(&go_hook.HookConfig{
+	Queue:        internal.Queue("discovery"),
```

Comment:
```suggestion
```
I think, it is worth keeping the hook in main queue.

Reply:
removed